tools/src/main/java/org/apache/kafka/tools/ClientCompatibilityTest.java

```diff
@@ -311,6 +297,18 @@ void testAdminClient() throws Throwable {
         }
     }
 
+    private boolean isFoundNewTopic(Collection<TopicListing> listings, String newTopicName) {
```

Comment:
This should be called `topicExists` or something like that. `newTopicName` then becomes `topicName`. The variables inside the method should also be changed and the exception should include the variable, not a hardcoded value.